jung-visualization/src/main/java/edu/uci/ics/jung/visualization/BaseVisualizationModel.java

```diff
@@ -168,11 +168,6 @@ public void setLayoutAlgorithm(LayoutAlgorithm<N, Point2D> layoutAlgorithm) {
     layoutModel.accept(layoutAlgorithm);
   }
 
-  @Override
-  public void setLayoutSize(Dimension size) {
```

Comment:
So how do we manage changing the size of the window in which the visualization is being done?  More generally, do we have a mechanism for changing the dimensions of the area that we're using for layout?

Reply:
The LayoutModel is actually infinite in size. If you pick a node and drag it off somewhere, the layoutModel's size (and the spatialquadtree) updates to the new size large enough to hold everything. You can even see it happen in a couple of demo programs. There's also a LayoutModel::setSize(w,h) which replaced the old Layout::setSize(Dimension) that does it programmatically.
As far as the window (view) goes, you grab the corner with the mouse and resize it. :)

Reply:
It was the programmatic aspect that I was concerned with, yes.  :)

I hadn't seen the LayoutModel::setSize method; that's obviously a piece that we need.

Reply:
I put back the component listener in AddNodeDemo, so users can see exactly how to make the layout size change when the view size changes. We used to assume that it should always be that way, by controlling layout algorithm initiation via the setSize method. We no longer make that assumption on the user's part, so the user must explicitly state what should happen.